### PR TITLE
Add TrimLast to odin

### PR DESCRIPTION
**Pull request by ifosch**
This PR adds `trimLast` to odin_test.
This function cuts out the last character in a string.

### CODE REVIEW

File: mcc/odin/odin/odin.go [hunk 1]
```diff
@@ -13,6 +13,15 @@ import (
 // Duration specified time to wait for instance to be available.
 var Duration = time.Duration(5) * time.Second
 
+// TrimLast trims last character from a given string and returns the trimmed
+// one.
+func TrimLast(in string) string {
```
Comment (miguelbernadi) [inline mcc/odin/odin/odin.go]: Would be better not to have it exported as its functionality is completely unrelated to odin's.

Comment (ifosch) [reply mcc/odin/odin/odin.go]: Ok, I guess that if this is needed somewhere else, we will move to a separate library, then. ;)

Comment (ifosch) [reply mcc/odin/odin/odin.go]: @miguelbernadi in order to have this working I made it unexported, but moved to `odin_test` package.

**Review by miguelbernadi — APPROVED** [review]
+1

**Pull request merged**